releases/schedule.md

```diff
@@ -10,7 +10,7 @@ permalink: /releases/schedule/
 | [SDK3]        | February 5, 2018  |
 | **[SDK4]**    | May 4, 2018       | 3.28
 | [SDK5]        | August 2, 2018    |
```

Comment:
Just a drive-by suggestion: you should un-bold SDK4 and bold SDK5, since it is the current version.